package.json

```diff
@@ -5,7 +5,27 @@
   "main": "index.js",
   "dependencies": {
     "babel-plugin-transform-es3-property-literals": "^6.3.13",
-    "babel-preset-es2015": "^6.3.13",
+    "babel-plugin-check-es2015-constants": "^6.5.0",
```

Comment:
Keeping this list in sync with the babel-preset-es2015 module manually is not something I want to do. Is there a reason you can't upgrade to npm3?